src/components/InputFields/InputFields.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const buildInputField = ({ type, value, handleChange }) => {
```

Comment:
I think this could work as a function in a helper file that we call from the Login and SignUp components.

Reply:
@kaohman that's a great idea! I tried to do that initially but it didn't work well, but was able to update it... would you mind looking over the newest version and see if that works for what you were thinking?